reference/resources/kubecontrollersconfig.md

```diff
@@ -0,0 +1,139 @@
+
+
+#### Spec
+
+| Field                  | Description                                               | Accepted Values                    | Schema | Default    |
+|------------------------|-----------------------------------------------------------|------------------------------------|--------|------------|
+| logSeverityScreen      | The log severity above which logs are sent to the stdout. | Debug, Info, Warning, Error, Fatal | string | Info       |
+| healthChecks           | Enable support for health checks                          | Enabled, Disabled                  | string | Enabled    |
+| etcdv3CompactionPeriod | The period between etcdv3 compaction requests. Only applies when using etcd as the {{site.prodname}} datastore. | Set to 0 to disable, > 0 to enable |  [Duration string][parse-duration] | 10m |
+| controllers            | Enabled controllers and their settings                    |                                    | [Controllers](#controllers) | |
+
+#### Controllers
+
+| Field            | Description                                           |  Schema                                        |
+|------------------|-------------------------------------------------------|------------------------------------------------|
+| node             | Enable and configure the node controller              | omit to disable, or [NodeController](#nodecontroller) |
+| policy           | Enable and configure the network policy controller    | omit to disable, or [PolicyController](#policycontroller)           |
+| workloadEndpoint | Enable and configure the workload endpoint controller | omit to disable, or [WorkloadEndpointController](#workloadendpointcontroller) |
+| serviceAccout    | Enable and configure the service account controller   | omit to disable, or [ServiceAccountController](#serviceaccountcontroller)  |
+| namespace        | Enable and configure the namespace controller         | omit to disable, or [NamespaceController](#namespacecontroller)        |
+
+#### NodeController
+
+The node controller automatically cleans up configuration for nodes that no longer exist. Optionally, it can create host endpoints for all Kubernetes nodes.
+
+| Field                              | Description                 | Accepted Values   | Schema | Default    |
+|------------------------------------|-----------------------------|-------------------|--------|------------|
+| reconcilerPeriod | Period to perform reconciliation with the {{site.prodname}} datastore | | [Duration string][parse-duration] | 5m |
+| numberOfWorkers | Number of goroutines to allocate to the controller | > 0 | int | 1 |
+| syncLabels | When enabled, Kubernetes node labels will be copied to {{site.prodname}} node objects. | Enabled, Disabled | string | Enabled |
+| hostEndpoint | Controls allocation of host endpoints | | [HostEndpoints](#hostendpoints) | |
+
+#### HostEndpoints
```

Comment:
Maybe `HostEndpoint`? It doesn't read as nice but could be a point of confusion (someone might try to use `hostEndpoints`)

Reply:
Thanks; fixed.